drift/lib/src/runtime/manager/references.dart

```diff
@@ -0,0 +1,224 @@
+part of 'manager.dart';
+
+/// A simple function for generating aliases for referenced columns
+///
+/// This function is used internally by generated code and should not be used directly.
+// ignore: non_constant_identifier_names
+String $_aliasNameGenerator(
```

Comment:
It's used on a static field, so it can't be a private value on the class

```dart
class BaseFoo {
  func();
}

class Foo extends BaseFoo {
  static const name = func(); // Doesnt work
}
```